docs/ancestry.md

```diff
@@ -675,8 +656,48 @@ is {math}`L (1 - e^{-r})`
 and so {math}`Lr (1 - r/2) < L (1 - e^{-r}) < Lr`.
 :::
 
-For example, the following defines a recombination map for three chromosomes
-each 1 centiMorgan in length:
+(sec_ancestry_multiple_chromosomes)=
+
+### Multiple chromosomes
+
+Msprime does not directly support simulating multiple chromosomes
+simultaneously, but we can emulate it using a single linear genome split into
+chromosome segments. Multiple chromosomes are modelled by specifying
+a recombination map with single base-pair segments with recombination
+probability 1/2 separating adjacent chromosomes. The probability that
+a recombination breakpoint occurs at a given base pair that has recombination
+rate {math}`r` is {math}`1-e^{-r}`. Thus, we set the recombination rate in the
```

Comment:
"The probability that a recombination breakpoint occurs *over one unit of time*"?